### PR TITLE
Docs, Contributing, Readme draft

### PR DESCRIPTION
**Pull request by okuchaiev**
This PRs kickstarts the following:

- [x] Overall RTD structure
- [x] Contributing guidlines
- [x] README.rst 

Signed-off-by: Oleksii Kuchaiev <okuchaiev@nvidia.com>

### CODE REVIEW

**Comment by okuchaiev** [comment]
note, that I expect readme to be significantly re-worked prior to any "launch"

File: docs/sources/Makefile [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+	@echo "Build finished; now you can run HTML Help Workshop with the" \
+	      ".hhp project file in $(BUILDDIR)/htmlhelp."
+
+.PHONY: qthelp
+qthelp:
+	$(SPHINXBUILD) -b qthelp $(ALLSPHINXOPTS) $(BUILDDIR)/qthelp
+	@echo
+	@echo "Build finished; now you can run "qcollectiongenerator" with the" \
+	      ".qhcp project file in $(BUILDDIR)/qthelp, like this:"
+	@echo "# qcollectiongenerator $(BUILDDIR)/qthelp/OpenSeq2Seq.qhcp"
```
Comment (tkornuta-nvidia) [inline docs/sources/Makefile:97]: do we need this?

Comment (okuchaiev) [reply docs/sources/Makefile:97]: probably not. I think we only need "help" target for RTD, no? Or RTD/Sphynx needs some other targets too? (I am not sure)

Comment (okuchaiev) [reply docs/sources/Makefile:97]: basically the minimal version for RTD would be ideal

File: .readthedocs.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# =============================================================================
```
Comment (tkornuta-nvidia) [inline .readthedocs.yml:1]: decorator

File: README.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
+ 
+|status| |license| |lgtm_grade| |lgtm_alerts| |black|
+
+.. |status| image:: http://www.repostatus.org/badges/latest/active.svg
```
Comment (tkornuta-nvidia) [inline README.rst:4]: the badges should indicate 'candidate' branch not latest...

File: docs/sources/Makefile [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+	@echo "  man        to make manual pages"
+	@echo "  texinfo    to make Texinfo files"
+	@echo "  info       to make Texinfo files and run them through makeinfo"
+	@echo "  gettext    to make PO message catalogs"
+	@echo "  changes    to make an overview of all changed/added/deprecated items"
+	@echo "  xml        to make Docutils-native XML files"
+	@echo "  pseudoxml  to make pseudoxml-XML files for display purposes"
+	@echo "  linkcheck  to check all external links for integrity"
+	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
+	@echo "  coverage   to run coverage check of the documentation (if enabled)"
```
Comment (tkornuta-nvidia) [inline docs/sources/Makefile:48]: targets need to be cleaned

File: docs/sources/Makefile (hunk 1 above)
Comment (tkornuta-nvidia) [reply docs/sources/Makefile:97]: ok

File: docs/sources/source/conf.py [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+    'dateutil',
+    'wget',
+    'scipy',
+    'pandas',
+    'matplotlib',
+    'sklearn',
+    'braceexpand',
+    'webdataset',
+    'tqdm',
+    'numba',
```
Comment (tkornuta-nvidia) [inline docs/sources/source/conf.py:62]: PL? hydra?

**Pull request merged**